Source/Core/Core/Movie.cpp

```diff
@@ -686,6 +686,11 @@ static void SetInputDisplayString(ControllerState padState, int controllerID)
   display_str += Analog2DToString(padState.AnalogStickX, padState.AnalogStickY, " ANA");
   display_str += Analog2DToString(padState.CStickX, padState.CStickY, " C");
 
+  if (padState.IsConnected)
+    display_str += " CONNECTED";
+  else
+    display_str += " DISCONNECTED";
```

Comment:
Maybe we should just replace the entire string with "DISCONNECTED" if it's disconnected? Or is there some other information that actually is relevant to display when the controller is disconnected?

We should at least be able to skip showing "CONNECTED" when it's connected. That's going to be the case very often, so specifically saying it just feels like a waste of space to me.